frontend/packages/ux-editor/src/components/toolbar/ToolbarItemComponent.module.css

```diff
@@ -10,7 +10,9 @@
 
 .componentIcon {
   min-width: 40px;
-  flex: 0 0 32px;
+  display: flex;
+  font-size: 2rem;
+  flex: 0 0 40;
 }
```

Comment:
Her tror jeg også vi kan gå frem litt annerledes. Vi trenge r ikke overstyre bredden til ikonet her. `display: flex;` og `font-size: 2rem;` burde holde her, så kan vi sette `gap: .5rem` på `.toolbarItem` for å få avstand.